guby_backend/chat.py

```diff
@@ -2,32 +2,33 @@
 import os
 import logging
 import traceback
+import sys
 
 logger = logging.getLogger(__name__)
 
-rocket_admin = os.environ['ROCKET_ADMIN_USERNAME']
-rocket_password = os.environ['ROCKET_ADMIN_PASSWORD']
-rocket_host = os.environ['ROCKET_SERVER_HOST']
-rocket_port = os.environ['ROCKET_SERVER_PORT']
-server_url = 'http://' + rocket_host + ':' + rocket_port
-rocket = RocketChat(rocket_admin, rocket_password, server_url=server_url)
-
 
 def create_channel(channel_name: str) -> dict:
     '''
     This method takes channel name as input and returns a dict which contains messages and information regarding channel creation
     Use the dict['success'] to determine the channel creation
     '''
+    rocket_admin = os.environ['ROCKET_ADMIN_USERNAME']
+    rocket_password = os.environ['ROCKET_ADMIN_PASSWORD']
+    rocket_host = os.environ['ROCKET_SERVER_HOST']
+    rocket_port = os.environ['ROCKET_SERVER_PORT']
+    server_url = 'http://' + rocket_host + ':' + rocket_port
```

Comment:
For string concatenation, let us follow the [f-string](https://realpython.com/python-f-strings/) approach everywhere? IIRC for group creation, @raghothams follows the same thing

```suggestion
    server_url = f'http://{rocket_host}:{rocket_port}'
```